modules/liveIntentIdSystem.js

```diff
@@ -1,387 +1,14 @@
-/**
```

Comment:
hmm, it's a bit unfortunate that it doesn't show the diff between this one and /libraries/liveIntentIdSystem/liveIntentIdSystem.js (treating it as a move), as that makes this a bit more difficult to review. No suggestions how to fix this though.